### ISSUE TITLE
Remnant default pref in "2019 preview" builds

### ISSUE THREAD

**Issue by Vangelis66** [Aris-t2/ClassicThemeRestorer#419]
In the **1.7.8.2019.xx.xx** builds of CTR, targeting mainly (Classic) Waterfox 56 and official Basilisk 52, 

> removed the option "add-on manager version number"
> Basilisk 2019 and Waterfox don't need this option, because version number is active by default

However, the corresponding (default) pref has been left behind: 

https://github.com/Aris-t2/ClassicThemeRestorer/blob/a3c25750b24a3b73b712e60aebcb16f702866fb8/xpi/defaults/preferences/options.js#L91

but, instead, with a value of `false` ; the value itself (true | false) is irrelevant here, as the GUI option has been removed from CTR (as per **release notes**); shouldn't the same apply to the pref itself?

Unless, OTOH, this is still needed to support Firefox v52-56; in any case, it shouldn't apply to Basilisk (because it _is_ present inside Basilisk's `about:config` tab), should it? ...

Apologies in advance if I have figured things the wrong way... 
Best regards 😄

**Comment by Aris-t2** [comment]
You are right, the pref can be removed as it does not even get monitored but CTRs observers anymore. It does no harm though.
